tests/h/exceptions_test.py

```diff
@@ -49,12 +49,7 @@ def test_it_sets_type_and_message(self):
     def test_it_sets_default_response_code(self):
         exc = exceptions.OAuthTokenError("boom", "mytype")
 
-        assert exc.status_code == 400
-
-    def test_it_sets_custom_response_code(self):
-        exc = exceptions.OAuthTokenError("boom", "mytype", 500)
-
-        assert exc.status_code == 500
```

Comment:
This is the unused custom status code feature that was removed from `OAuthTokenError`.